dm/pkg/conn/basedb.go

```diff
@@ -232,17 +278,27 @@ func (d *BaseDB) DoTxWithRetry(tctx *tcontext.Context, queries []string, args []
 	return err
 }
 
-// CloseBaseConn release BaseConn resource from BaseDB, and close BaseConn.
-func (d *BaseDB) CloseBaseConn(conn *BaseConn) error {
+// CloseConn release BaseConn resource from BaseDB, and returns the connection to the connection pool,
+// has the same meaning of sql.Conn.Close
+func (d *BaseDB) CloseConn(conn *BaseConn) error {
```

Comment:
this `close` has the same meaning as sql.Conn.Close, i.e. put it into connection pool.